lldb/include/lldb/Core/Progress.h

```diff
@@ -12,6 +12,7 @@
 #include "lldb/Utility/ConstString.h"
 #include "lldb/lldb-types.h"
 #include <atomic>
+#include <map>
```

Comment:
Unless you need the sorted iteration property you probably want to use something more efficient instead.

Reply:
I shouldn't need a sorted map so I'll go with StringMap for the updated patch with the class that will handle the progress report map. 👍🏾